includes/tools.php

```diff
@@ -103,7 +103,10 @@ function render_tools_page() {
 			<p><?php esc_html_e( 'Airstory also uses Secure Socket Layers (SSL) to send data back and forth between your site and Airstory, so your site must be accessible over HTTPS. Fortunately, SSL certificates can be generated by your host, typically at no cost to you.', 'airstory' ); ?></p>
 		<?php endif; ?>
 
-
+		<h2><?php esc_html_e( 'Get Support', 'airstory' ); ?></h2>
+		<p><?php echo wp_kses_post( 'The Airstory WordPress plugin is backed by <a href="https://www.liquidweb.com/support/heroic-promise.html">Liquid Web\'s Heroic Support</a>; if you run into any issues when working with the Airstory plugin, please <a href="https://wordpress.org/support/plugin/airstory" target="_blank">feel free to open an issue in the WordPress.org plugin repo</a>.', 'airstory' ); ?></p>
```

Comment:
This feels like it should be wrapped in an `sprintf` to make the translation easier in the long term.